community/code-of-conduct.md

```diff
@@ -0,0 +1,166 @@
+
+1. If it has happened for the first time, dm the person. And let {pronoun} know that this behaviour is offensive to the subject. If not comfortable confronting then **????????????? @team comments**
+2. Post on a public forum that the subject finds this particular behaviour offensive/xyz. Name calling doesn't help.
+3. If it's repetitive, **what to do ???????????????????? @team , comments**
+
+## Basic Protocol in case you are accused of harassment/bullying/offending/doing weird shits/unacceptable behaviour
+
+Subject -> Who has been accused
+
+1. Well the subject can start by saying sorry, and ensuring that similar behaviour is not repeated in front of the person who has accused the subject "ever". If it's ingrained in the subject’s habit let the accussee know that, it's in your habit you will make sincere effort to decrease the frequency. The subject can take a note and share updates of progress {pronoun} have made on that person's feedback.
+2. Adding "genuinely" prefix in front of sorry doesn't help.
+3. It doesn't matter if the subject’s sorry or change is genuine or fake, as long as it's change it's good.
+
+## Basic Protocol in case you have noticed harassment.
+
+**@team comments**
+
+
+
+## Basic Protocol in case you are an executive/advisor and have to take action
+
+**@team comments**
+
+## What is Considered as Sexual Harassment at Work Place ?
+
+What workplace at koss includes, **@team ????????????**
+
+Sexual harassment includes many things but is not limited to
+
+[Ref](https://www.un.org/womenwatch/osagi/pdf/whatissh.pdf)
+
+1. Actual or attempted rape or sexual assault.
+2. Unwanted pressure for sexual favors.
+3. Unwanted deliberate touching, leaning over, cornering, or pinching.
+4. Unwanted sexual looks or gestures.
+5. Unwanted letters, telephone calls, or materials of a sexual nature.
+6. Unwanted pressure for dates.
+7. Unwanted sexual teasing, jokes, remarks, or questions.
+8. Referring to an adult as a girl, hunk, doll, babe, or honey.
+9. Whistling at someone.
+10. Cat calls.
+11. Sexual comments.
+12. Turning work discussions to sexual topics.
+13. Sexual innuendos or stories.
+14. Asking about sexual fantasies, preferences, or history.
+15. Personal questions about social or sexual life.
+16. Sexual comments about a person's clothing, anatomy, or looks.
+17. Kissing sounds, howling, and smacking lips.
+18. Telling lies or spreading rumors about a person's personal sex life.
+19. Neck massage.
+20. Touching an employee's clothing, hair, or body.
+21. Giving personal gifts., Hanging around a person.
+22. Hugging, kissing, patting, or stroking.
+23. Touching or rubbing oneself sexually around another person.
+24. Standing close or brushing up against a person.
+25. Looking a person up and down (elevator eyes).
+26. Staring at someone.
+27. Sexually suggestive signals.
+28. Facial expressions, winking, throwing kisses, or licking lips.
+29. Making sexual gestures with hands or through body movements.
```

Comment:
A lot of the points in this list seem repetitive, the shorter, more general the list will be the easier it will be to use.

Reply:
How is this ?
| Physical | Verbal | Non-Verbal |
|--------- | ------ | ---------  |
| Touching the person's clothing, hair, or body | Referring to an adult as a girl, hunk, doll, babe, or honey | Looking a person up and down (Elevator eyes) |
| Hugging, kissing, patting, or stroking | Whistling at someone, cat calls | Staring at someone |
| Touching or rubbing oneself sexually around another person | Making sexual comments about a person's body , innuendos , Turning work discussions to sexual topics | Blocking a person's path |
| Standing close or brushing up against another person | Telling sexual jokes or stories | Following the person |
| Actual or attempted rape or sexual assault | Asking personal questions about social or sexual life | Giving personal gifts |
| Unwanted deliberate touching, leaning over, cornering, or pinching | Making kissing sounds, howling, and smacking lips | Displaying sexually suggestive visuals | 
| | Making sexual comments about a person's clothing, anatomy, or looks | Making sexual gestures with hands or through body movements
| | Repeatedly asking out a person who is not interested | Making facial expressions such as winking, throwing kisses, or licking lips
| | Telling lies or spreading rumors about a person's personal sex life |

Reply:
This one is quite nicer!

Reply:
can point of obscene calling/texting or objectionable(details can be added) messaging be added??

Reply:
Oh, ha! That is also a form of harassment. @rava-dosa Can we add it as well, please?